backend/src/components/school.js

```diff
@@ -19,6 +29,128 @@ async function getSchoolBySchoolID(req, res) {
   }
 }
 
+async function getSchoolLastUpdated(req, res) {
```

Comment:
This function is identical to getSchoolBySchoolID above. Maybe use that function instead?